starter/js/atm.js

```diff
@@ -1,12 +1,45 @@
 //Begin with the document ready function
+$(document).ready(function() {
 
-    //Checking account deposit function
 
+    //Checking account deposit function
       //On click of the depositChecking button
+    $('#depositChecking').click(function(event) {
+      // console.log(event);
+            //Get value from the amountChecking input field
+        var depositAmount = $('#amountChecking').val();
+        // console.log(depositAmount)
+        //Take that value and add it to the existing value in the checkingBalance div
+        var checkingBal = $('#checkingBalance').text();
+        // console.log(checkingBal);
+
+        // console.log(strValue);
+        var totalBalance =  parseInt(depositAmount) + parseInt(checkingBal.replace('$',''));
+        $('#checkingBalance').text('$' + totalBalance);
+        // console.log(totalBalance);
+
+        });
+          $('#withdrawChecking').click(function(event) {
+      // console.log(event);
+            //Get value from the amountChecking input field
+        var withdrawAmount = $('#amountChecking').val();
+        // console.log(depositAmount)
+        //Take that value and add it to the existing value in the checkingBalance div
+        var checkingBal = $('#checkingBalance').text();
+        // console.log(checkingBal);
 
-        //Get value from the amountChecking input field
+        // console.log(strValue);
+        var totalBalance = parseInt(checkingBal.replace('$','')) - parseInt(withdrawAmount);
```

Comment:
This parseInt isn't working on the checkingBal because it's trying to convert with the $ before replacing it with the empty string. You can fix this by doing `var checkingBal = $('#checkingBalance').text().replace('$','');` above on line 28 and then you will get a valid totalBalance.